### ISSUE TITLE
Gray background bar/canvas

### ISSUE THREAD

**Issue by HansenChristoffer** [B00merang-Project/Windows-XP#45]
Hello, awesome work so far, really liking it!

My main issue is with the Arc Menu in Gnome. I am on Manjaro if that matters and I am using Windows XP Luna.
Nothing changes the look of the Arc menu and at the bottom of the Arc Menu, there is a gray background canvas. It isn't just the Arc Menu, I can see it behind the extension "Pamac available update". I will post pictures below for reference.

I am not certain what is causing this. Other themes I have downloaded including default ones, does not have the gray canvas in the background.

### **Arc Menu**
![image](https://github.com/B00merang-Project/Windows-XP/assets/57729355/9840719f-6783-4870-bdaa-29b017b4585c)

### **Pamac Available update**
![image](https://github.com/B00merang-Project/Windows-XP/assets/57729355/900c401d-63ee-4be0-98fc-77546dde4e5a)

### **Appearance**
![image](https://github.com/B00merang-Project/Windows-XP/assets/57729355/02c4ae25-cc6e-41cc-8c61-093cebe87bbb)

**Comment by Elbullazul** [comment]
It's caused by some changes that happened to the way gnome-shell handles  themes.

I would like to rebase the shell themes on a more durable codebase, but I haven't spent much time on this project lately.

I'll see if I can push a commit that fixes this in the coming days

**Comment by Elbullazul** [comment]
no, the shell themes are a different type of theme that are found in the `gnome-shell` folder.

a few versions ago, the default shell theme changed and I haven't had time to update my shell themes, so some parts of the default shell theme are showing (the grey patches and the shadows in menus).

I'll need to add overrides for those cases
